src/libPMacc/CMakeLists.txt

```diff
@@ -23,11 +23,26 @@
 ################################################################################
 # PMacc tests
 ################################################################################
-cmake_minimum_required(VERSION 3.3)
-project ("PMacc-tests")
+cmake_minimum_required(VERSION 3.1)
```

Comment:
This is now the same as PIConGPU. I don't see a reason, why we should use a higher requirement here. `CMAKE_CXX_STANDARD` is not an issue, as the correct value is ensured (the same way as in PIConGPU).
The higher versions just makes it harder to use this with a default-configured system.